Assets/Scripts/BalanceMode.cs

```diff
@@ -0,0 +1,25 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BalanceMode : GameMode {
+    const string NAME = "Balance Mode";
+    GameObject boardPrefab = (GameObject) Resources.Load("BalanceBoard", typeof(GameObject)); 
+
+    /**
+        Get the prefab from the game mode
+    */
+    public override GameObject getBoardPrefab() {
+        return boardPrefab;
+    }
+
+    /**
+        Get name of game mode
+    */
+    public override string getName() {
+        return NAME;
+    }
+}
+
+
+// "Prefabs/BalanceBoard.prefab"
```

Comment:
I'm unsure if we're gonna care about "no blank line" at end of files "errors"?

Reply:
Might as well have one blank line at the end. However that last line was a comment I forgot to remove, so that needs to go regardeless ;)

Reply:
Onthe other hand, it doesn't seem like I am able to see whether or not there are blank lines at the end of files, so please point it out ot me if I forget...